src/web.ts

```diff
@@ -0,0 +1,68 @@
+// Abstractions
+export * from './core/Billboard'
+export * from './core/Detailed'
+export * from './core/Line'
+export * from './core/PositionalAudio'
+export * from './core/Text'
+export * from './core/Environment'
+export * from './core/Effects'
+
+// Cameras
+export * from './core/OrthographicCamera'
+export * from './core/PerspectiveCamera'
+
+// Controls
+export * from './core/DeviceOrientationControls'
+export * from './core/FlyControls'
+export * from './core/MapControls'
+export * from './core/OrbitControls'
+export * from './core/TrackballControls'
+export * from './core/TransformControls'
+export * from './core/PointerLockControls'
+
+// Loaders
+export * from './core/Loader'
+export * from './core/useCubeTexture'
+export * from './core/useFBX'
+export * from './core/useGLTF'
+export * from './core/useProgress'
+export * from './core/useTexture'
+
+// Misc
+export * from './core/Html'
+export * from './core/meshBounds'
+export * from './core/Reflector'
+export * from './core/Shadow'
+export * from './core/Stats'
+export * from './core/useAspect'
+export * from './core/useCamera'
+export * from './core/useDetectGPU'
+export * from './core/useHelper'
+export * from './core/useContextBridge'
+export * from './core/useAnimations'
+export * from './core/useFBO'
+
+// Modifiers
+export * from './core/useSimplification'
+export * from './core/useSubdivision'
+export * from './core/useTessellation'
+export * from './core/useEdgeSplit'
+export * from './core/CurveModifier'
+
+// Shaders
+export * from './core/MeshDistortMaterial'
+export * from './core/MeshWobbleMaterial'
+export * from './core/shaderMaterial'
+export * from './core/Sky'
+export * from './core/softShadows'
+export * from './core/Stars'
+export * from './core/ContactShadows'
+
+// Shapes
+export * from './core/shapes'
+export * from './core/RoundedBox'
+export * from './core/ScreenQuad'
+
+// Prototyping
+export * from './core/useMatcapTexture'
+export * from './core/useNormalTexture'
```

Comment:
This could be a folder with the following structure
- web
- - index.ts
- - loader.tsx
- - html.tsx

We'd then remove `loader` and `html` from `core` and core could have a reliable `index` file.